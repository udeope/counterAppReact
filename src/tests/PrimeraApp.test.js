import React from 'react';
import { getByText, render } from "@testing-library/react";
import PrimeraApp from "../PrimeraApp";
import { shallow } from 'enzyme';


describe('Pruenas en PrimeraApp', () => {
    
    // test('debe mostrar el mensaje "Hola soy Goku"', () => {
    //     const saludo = 'Hola soy Goku';
    //     const { getByText } = render( <PrimeraApp saludo={saludo} /> )
    //     //wrapper.getByText()
    //     expect( getByText(saludo)).toBeInTheDocument();
    // })
    
    test('debe mostrar PrimeraApp correctamente', () => {
        
        const saludo = 'Hola soy Goku';
        const wrapper = shallow( <PrimeraApp saludo={saludo} />);

        expect( wrapper ).toMatchSnapshot();

    });

    test('debe mostrar el subtitulo enviado por props', () => {
        
        const saludo = 'Hola soy Goku';
        const subtitulo = 'Soy un subtitle';

        const wrapper = shallow( <PrimeraApp saludo={saludo} subtitulo={subtitulo} />);

        const textoParrafo = wrapper.find('p').text();
        console.log(textoParrafo);

        expect(textoParrafo).toBe(subtitulo)
        


    })
    
    


});