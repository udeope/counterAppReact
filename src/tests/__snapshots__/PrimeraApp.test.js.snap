// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruenas en PrimeraApp debe mostrar PrimeraApp correctamente 1`] = `
<Fragment>
  <h1>
    Hola soy Goku
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
